cog/perk.py

```
@@ -0,0 +1,321 @@
+                )
+                return
+
+        # 先告知 Discord 稍等，避免資料庫或 API 較慢時超過 3 秒限制
+        await ctx.defer()
+
+        user_id = ctx.author.id
+        connection, cursor = link_sql()
+        point = read(user_id, "point", cursor)
+        if point < ROLE_PRICE:
+            await ctx.respond(
+                f"電電點不足！需要 {ROLE_PRICE} 點，你目前只有 {point} 點。"
```

Ruff (0.15.20)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)